### PR TITLE
Fix window launch glitch

### PR DESCRIPTION
**Pull request by mhutch**
Fix glitches on Windows where the window would resize immediately after startup. Also cleans up some dead code.

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/DefaultWorkbench.cs [hunk 1]
```diff
@@ -673,7 +670,6 @@ public Properties GetStoredMemento (ViewContent content)
 						Iconify ();
 					}
 					//GdkWindow.State = memento.WindowState;
-					FullScreen = memento.FullScreen;
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/DefaultWorkbench.cs]: Can't we make this work on Mac?

Comment (Therzok) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/DefaultWorkbench.cs]: It should work on mac.

Comment (Therzok) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/DefaultWorkbench.cs]: Also, agreed, maybe we should not do this on windows only.

**Comment by mhutch** [comment]
Is restoring fullscreen state reasonable on Mac?

The code was broken already because it didn't save the fullscreen state, it only restored it.

**Comment by slluis** [comment]
Xcode saves and restores full screen mode, so I guess it is reasonable to do it.

**Comment by mhutch** [comment]
Ah ok! I didn't have a Mac handy to check.

**Pull request merged**